src/platforms/rcore_desktop.c

```diff
@@ -996,6 +996,18 @@ void DisableCursor(void)
     CORE.Input.Mouse.cursorHidden = true;
 }
 
+// Start the frame
+void StartFrame(void)
+{
+    TRACELOG(LOG_WARNING, "StartFrame() not implemented on target platform");
```

Comment:
I haven't tested this but wouldn't this cause a lot of warnings spam in the console/logs in debug mode?
Probable just leave the function body empty and just add a comment instead.